torch/nn/qat/modules/conv.py

```diff
@@ -174,4 +186,16 @@ def to_float(self):
         conv.weight = torch.nn.Parameter(self.weight.detach())
         if self.bias is not None:
             conv.bias = torch.nn.Parameter(self.bias.detach())
+        cls = type(self)
+        # conv relu
+        if issubclass(cls, _FusedModule):
+            modules = [conv]
+            assert hasattr(cls, "_FLOAT_RELU_MODULE")
+            relu = cls._FLOAT_RELU_MODULE()  # type: ignore[attr-defined]
+            modules.append(relu)
+            fused = cls._FLOAT_MODULE(*modules)  # type: ignore[arg-type]
+            fused.train(self.training)
+            return fused
+        else:
+            return conv
         return conv
```

Comment:
nit: extra return statement here, same in L87

Reply:
oh sorry, will remove